api/filter/http/fault.proto

```diff
@@ -5,17 +5,23 @@ package envoy.api.v2.filter.http;
 import "api/rds.proto";
 import "api/filter/fault.proto";
 
+import "validate/validate.proto";
+
 // Abort specification is used to prematurely abort a HTTP/gRPC/Mongo/Redis
 // operation/TCP connection with a pre-specified error code.
 message FaultAbort {
   // An integer between 0-100 indicating the percentage of requests/operations/connections
   // that will be aborted with the error code provided.
-  uint32 percent = 1;
+  uint32 percent = 1 [(validate.rules).uint32.lte = 100];
 
   // Applicable only for HTTP connections.
   oneof error_type {
+    option (validate.required) = true;
     // HTTP status code to use to abort the HTTP request.
-    uint32 http_status = 2;
+    uint32 http_status = 2 [(validate.rules).uint32 = {
+        gte: 200,
+        lte: 600
```

Comment:
@rshriram this should be less than 600 (lte 599), sorry didn't catch this before.